recipe/meta.yaml

```diff
@@ -15,7 +15,7 @@ package:
 
 source:
   url: https://github.com/esmf-org/esmf/archive/refs/tags/v{{ version }}.tar.gz
-  sha256: dc270dcba1c0b317f5c9c6a32ab334cb79468dda283d1e395d98ed2a22866364
+  sha256: d7ab266e2af8c8b230721d4df59e61aa03c612a95cc39c07a2d5695746f21f56
   patches:
     # remove these very long running tests only for aarch64 nompi (which is hanging)
     - remove-long-running-tests.patch  # [aarch64 and mpi == 'nompi']
```

Comment:
It looks like we might need to add `linux-64` and `ppc64le` with `mpich`.  These now also seem to be hanging.

Reply:
Takes awhile but at least linux-64 is moving again.

Reply:
Looks like we just need to be patient.